main-app/client/src/get_uri.js

```diff
@@ -1 +1 @@
-export const API_ENDPOINT = 'http://192.168.99.100:4000';
+export const API_ENDPOINT = 'http://localhost:4000';
```

Comment:
putting localhost as an endpoint actually makes it fails on my Windows laptop.

@mitchellwandrews  @tienyuan  How do you think we should structure this so we can toggle between the Windows and non-Windows environments?

Reply:
@serpient if you use 127.0.0.1 does that work?

Reply:
@mitchellwandrews `127.0.0.1` returns errors

Reply:
Maybe it can be bumped into a env variable that can be read from the docker config?